lib/MusicBrainz/Server/Data/Release.pm

```diff
@@ -1,13 +1,16 @@
 package MusicBrainz::Server::Data::Release;
 
+use 5.18.2;
```

Comment:
Why? (might make sense to specify in the commit message?)

Reply:
To use the [state](https://perldoc.perl.org/functions/state.html) function (I'll add a comment)